docs/_documentations/codewind-openapi-vscode.md

```diff
@@ -22,6 +22,21 @@ The Codewind OpenAPI Tools for VS Code provides commands that invoke the OpenAPI
 3. This extension pulls the [OpenAPI Generator CLI Docker Image](https://github.com/OpenAPITools/openapi-generator#16---docker) and runs the OpenAPI Generator in a Docker container. Install Docker if necessary.
 4. Install the Codewind OpenAPI Tools from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-openapi-tools) or by searching for `Codewind OpenAPI` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
 
+## Generating and building code in an existing Java Maven project
```

Comment:
I'd like this section to be for the Open API Tools for VS Code (NOT, the tools for eclipse, because the steps and workaround are different)

Reply:
@keithchong This PR update is for a document that only applies to VS Code tools. Eclipse would need to be a separate update. Is a title like "Generating and building code and clients for OpenAPI tools" suitable? Or is there a term that would cover both "code" and "clients" together?

Reply:
The title can be improved.  'Code' is too general and we need to mention 'server'.  Change the title from `Generating and building code and clients for OpenAPI tools` to something like:
`Generating and building client and server stubs using the OpenAPI tools`